examples/flutter_gallery/lib/demo/material/text_form_field_demo.dart

```diff
@@ -256,28 +267,35 @@ class TextFormFieldDemoState extends State<TextFormFieldDemo> {
                     ),
                     maxLines: 1,
                   ),
-                  const SizedBox(height: 24.0),
-                  PasswordField(
-                    fieldKey: _passwordFieldKey,
-                    helperText: 'No more than 8 characters.',
-                    labelText: 'Password *',
-                    onFieldSubmitted: (String value) {
-                      setState(() {
-                        person.password = value;
-                      });
-                    },
-                  ),
-                  const SizedBox(height: 24.0),
-                  TextFormField(
-                    enabled: person.password != null && person.password.isNotEmpty,
-                    decoration: const InputDecoration(
-                      border: UnderlineInputBorder(),
-                      filled: true,
-                      labelText: 'Re-type password',
+                  Autofill(
```

Comment:
Where is the autofillHints for this one? Or does it infer password automatically?

Reply:
Ah sorry. Forgot to add hints for the text fields.